### ISSUE TITLE
Exclude odd jobs experience from total experience

### ISSUE THREAD

**Issue by megamanxl** [preferit/cv#7]
For you talents one might want to list odd jobs that the person had for example during university studies. But if the job is not software relatable, the experience should no be included in the total experience

**Comment by gregoryv** [comment]
I agree that one might want to exclude some experiences from the CV for whatever reason.

## Alternative 1 - remove the item from the CV altogether

Then it doesn't show up in the experience and is not part of the calculation

## Alternative 2 - filter out experiences

Somewhat as the PR #6 , this results in the experience being listed but not counted in the total experience. If we do this then the years of experience is misleading. Eg. if you show three entries (1 year each) and two of them are not counted, then experience years is showing 1, but the reader can clearly see there are three entries of 1 year each. 

If we filter experiences I think they should not be visible at all, but that will lead to "missing" years which is probably Not what we want.

## Alternative 3 - extended experience value

Currently the experience means, number of working years. It does not distinguish between (experience of X) within those years. We could however add/extend the experience. E.g.

- Number of years as the role Developer(using the roles section) or
- Number of years using language X (using the tags)

Then it's a matter of trying to place that information in the resulting document in an appropriate manner.

## Alternative 4 - Augment technical experience section

Technical experience section is showing but it's a manually curated list. We could augment it somehow to show years of experience for each technology, again using the tags from the CV.

## Summary

If alternative 1 is not good enough for hiding experiences I think we should try to visualise "relevant" experience in some other manner. I've given some alternatives, maybe you can come up with couple more?

We should also keep presentation related information separate from the CV file as it's only purpose is to document ones history and let the tool filter using some input.

**Comment by megamanxl** [comment]
One alternative might be to not list the period. And have all experience with no listed period, be listed last in the entry list.
